pkg/util/elasticsearch/elasticsearch.go

```diff
@@ -68,11 +70,15 @@ func NewClient(cfg config.ElasticSearch) (Client, error) {
 }
 
 func (c *client) Request(httpMethod, rawPath string, payload io.Reader) ([]byte, error) {
-	esURL, err := c.parseUrl(rawPath)
+	return c.RequestWithCtx(context.Background(), httpMethod, rawPath, payload)
```

Comment:
this context should be canceled when the request is done
```suggestion
         ctx := context.Background()
         defer ctx.Done()
	return c.RequestWithCtx(ctx, httpMethod, rawPath, payload)
```

Reply:
Oh right, thanks for spotting, fixed.